### PR TITLE
Modulalrize dev docs

### PR DESCRIPTION
**Pull request by jayunit100**
quick refactor of dev docs, mostly copy and paste - to avoid staring at the sun when reading CONTRIBUTING.adoc

### CODE REVIEW

**Comment by jayunit100** [comment]
fixes #9406
cc @smarterclayton @bparees @smarterclayton

**Comment by jayunit100** [comment]
looks like the build stalled but i doubt this actually broke anything, unless i violated a docs formatting policy or something.

**Comment by jayunit100** [comment]
(bump) is this ok to merge? looks like build flake....

**Comment by smarterclayton** [comment]
Leave root contributing.adoc with a table of contents and reference to the subdir.

File: contributing/CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
+= Contributing to OpenShift
```
Comment (smarterclayton) [inline contributing/CONTRIBUTING.adoc]: Belongs under docs, can be docs/CONTRIBUTING.adoc

Comment (smarterclayton) [inline contributing/CONTRIBUTING.adoc]: Doesn't have to be same case actually

**Comment by smarterclayton** [comment]
Two comments. All docs under docs/ and leave a ref back so existing links
aren't broken.

On Jun 27, 2016, at 5:39 PM, jay vyas <notifications@github.com> wrote:

(bump) is this ok to merge? looks like build flake....

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9436#issuecomment-228883996>, or mute
the thread
<https://github.com/notifications/unsubscribe/ABG_p9LiPqlBqj1j2RJ4mvaEOf4Jzrwqks5qQEMSgaJpZM4I5t_G>
.

**Pull request closed without merging**